### PR TITLE
Windows: Update ICU to 65.1

### LINKED ISSUE #65: vcpkg: Needs ICU 65 data package
We are currently behind with our vcpkg ICU patch at ICU59 and this needs an update before the next rebuild because the ICU59 DL URL changed.

Maybe a good time to update the library directly.

### PR DESCRIPTION
**Pull request by Ghabry**
Reapplied our ICU 61 patch to ICU 65.

Fix #65

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,10 +2,10 @@
 /*/*/
 
 # Except some Windows folders
-!/windows/icu-easyrpg/*
+!/windows/icu-easyrpg/
```
Comment (Ghabry) [inline .gitignore:5]: With the "*" I get a warning that the folder is ignored. Without the "*" it works

Comment (carstene1ns) [reply .gitignore:5]: I remember i had a similar problem, that is why i changed it here too:
https://github.com/EasyRPG/buildscripts/pull/55/files#diff-a084b794bc0759e7a6b77810e01874f2

**Comment by carstene1ns** [comment]
sadtrombone

```
Building package icu-easyrpg[core]:x64-windows-static...
-- Using cached C:/Jenkins/workspace/toolchain-windows/windows/vcpkg/downloads/icu4c-65_1-src.tgz
-- Extracting source C:/Jenkins/workspace/toolchain-windows/windows/vcpkg/downloads/icu4c-65_1-src.tgz
-- Applying patch C:/Jenkins/workspace/toolchain-windows/windows/vcpkg/ports/icu-easyrpg/disable-escapestr-tool.patch
-- Applying patch C:/Jenkins/workspace/toolchain-windows/windows/vcpkg/ports/icu-easyrpg/remove-MD-from-configure.patch
-- Applying patch C:/Jenkins/workspace/toolchain-windows/windows/vcpkg/ports/icu-easyrpg/fix_parallel_build_on_windows.patch
CMake Error at scripts/cmake/vcpkg_apply_patches.cmake:51 (message):
  Applying patch failed.  error: corrupt patch at line 7

Call Stack (most recent call first):
  scripts/cmake/vcpkg_extract_source_archive_ex.cmake:134 (vcpkg_apply_patches)
  ports/icu-easyrpg/portfile.cmake:15 (vcpkg_extract_source_archive_ex)
  scripts/ports.cmake:90 (include)


Error: Building package icu-easyrpg:x64-windows-static failed with: BUILD_FAILED
```

**Comment by Ghabry** [comment]
Ah that is this autocrlf problem again. Is there some way to solve this on repository side? :/

Fixing this with a global git config in the meanwhile -_-

**Comment by carstene1ns** [comment]
yeah, this should work:

```ini
*.patch text eol=lf
```
or `-text`, to disable conversion

**Pull request merged**